08LessonEight/homework/exercise-08.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title></title>
+<head>
+   <link rel="stylesheet" href="css/exercise-08-style.css">
+  <title>The Box Model Homework Assignment</title>
+   <div class="parent-container">
+       <article>
+    <img src="img/truck.jpg">
+         <div>
+  <h4>SAFE & SECURE DELIVERY</h4>
+  Integer congue, elit non semper laoreet sed lectus orci posuh nisl tempor lacus felis ac mauris. Pellentesque in urna. Intege vitae felis vel magna posuere vestibulum.
+         <br> </br>
+     <button type="button" name="button">READ MORE</button>
+    </article>
+<article>
+    <img src="img/plane.jpg">
+  <h4>SAFE & SECURE DELIVERY</h4>
+  Integer congue, elit non semper laoreet sed lectus orci posuh nisl tempor lacus felis ac mauris. Pellentesque in urna. Intege vitae felis vel magna posuere vestibulum.
+    <br> </br>
+  <button type="button"name="button">READ MORE</button>
+</article>
+<article>
+    <img src="img/train.jpg">
+    <h4>SAFE & SECURE DELIVERY</h4>
+  Integer congue, elit non semper laoreet sed lectus orci posuh nisl tempor lacus felis ac mauris. Pellentesque in urna. Intege vitae felis vel magna posuere vestibulum.
+    <br> </br>
+  <button type="button"name="button">READ MORE</button>
+</div>
+</section>
+  </body>
```

Comment:
You matched what was expected on the assignment but there is still work to be done with your html code. I think we need to go over code readability. I also noticed that you are missing the closing `</html>` tag. Keep working hard and you will get there.